css-three-column-layout/styles.css

```diff
@@ -0,0 +1,49 @@
+.container {
+  max-width: 950px;
+  margin: 0 auto;
+  background-color: #1996da;
+  padding: 25px 50px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  padding: 0 10px;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  text-align: center;
+  align-items: center;
+}
+
+.circle {
+  width: 100px;
+  height: 100px;
+  background-color: #fff;
+  border-radius: 50%;
+  margin: 0 auto -20px;
+
+  /* Give the circle a margin top 0, auto for L/R to center it and -20px at the bottom to bring circle up */
+}
+
+.image-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
```

Comment:
try moving these into the `.circle` class (then you won't need the `image-container` element).
and to make sure the `.circle` is a circle, you can set a `width/height: 100px;` 
the margin can also be `0 auto`